tests/foreman/ui/test_registration.py

```diff
@@ -318,6 +326,7 @@ def test_global_registration_form_populate(
         cmd = session.host.get_register_command(
             {
                 'general.host_group': f'{hg_name}/{hg_nested_name}',
+                'general.activation_keys': module_ak_with_cv.name,
```

Comment:
AK is already updated in HG created above, so it is expected to be auto-populated here, can you remove this?